packages/edit-site/src/components/style-book/index.js

```diff
@@ -321,7 +334,102 @@ function StyleBook( {
 	);
 }
 
-const StyleBookBody = ( {
+/**
+ * Style Book Preview component renders the stylebook without the Editor dependency.
+ *
+ * @param {Object}   props            Component props.
+ * @param {string}   props.path       Path to the selected block.
+ * @param {Object}   props.userConfig User configuration.
+ * @param {Function} props.isSelected Function to check if a block is selected.
+ * @param {Function} props.onSelect   Function to select a block.
+ * @return {Object} Style Book Preview component.
+ */
+export const StyleBookPreview = ( {
+	path = '',
+	userConfig = {},
+	isSelected,
+	onSelect,
+} ) => {
+	const siteEditorSettings = useSelect(
+		( select ) => select( siteEditorStore ).getSettings(),
+		[]
+	);
+	// Update block editor settings because useMultipleOriginColorsAndGradients fetch colours from there.
+	dispatch( blockEditorStore ).updateSettings( siteEditorSettings );
+
+	const [ resizeObserver, sizes ] = useResizeObserver();
+	const { colors, gradients } = useMultipleOriginColorsAndGradients();
+
+	const [ duotones ] = useSettings( 'color.duotone.theme' );
+
+	const themeDuotone = duotones?.length
+		? [
+				{
+					name: _x(
+						'Theme',
+						'Indicates these duotone filters come from the theme.'
+					),
+					slug: 'theme',
+					duotones,
+				},
+		  ]
+		: null;
+
+	// Exclude the default colors and gradients.
+	const themeColors = colors?.filter( ( color ) => color.slug === 'theme' );
+	const themeGradients = gradients?.filter(
+		( gradient ) => gradient.slug === 'theme'
+	);
+
+	const examples = getExamples( {
+		colors: themeColors,
+		gradients: themeGradients,
+		duotones: themeDuotone || [],
+	} );
```

Comment:
```suggestion
	const colors = useMultiOriginPalettes();
	const examples = getExamples( colors );
```

If I understand correctly, we should be able to reuse the `useMultiOriginPalettes()` function.

Reply:
OH that works! Yes, thanks!